### DISCUSSION TITLE
What does the number in circular fingerprints refer to?

### DISCUSSION THREAD

**Discussion by apahl** [rdkit/rdkit#8009] [Q&A] [answered]
Hi, I would like to put this question out here, since we are having an internal discussion about it.  
Does the number in circular fingerprints like ECFC4 or ECFP6 refer to the radius or the diameter?  
KR Axel

**Comment by ptosco** [comment, marked as answer]
@apahl It is the diameter. See https://www.rdkit.org/docs/GettingStartedInPython.html at the paragraph
> When comparing the ECFP/FCFP fingerprints and the Morgan fingerprints generated by the RDKit, remember that the 4 in ECFP4 corresponds to the diameter of the atom environments considered, while the Morgan fingerprints take a radius parameter. So the examples above, with radius=2, are roughly equivalent to ECFP4 and FCFP4.

**Reply by apahl** [reply]
Thank you for the quick answer!
